cedar-policy-cli/Cargo.toml

```diff
@@ -11,10 +11,10 @@ homepage.workspace = true
 repository.workspace = true
 
 [dependencies]
-cedar-policy = { version = "=4.1.0", path = "../cedar-policy" }
-cedar-policy-formatter = { version = "=4.1.0", path = "../cedar-policy-formatter" }
-cedar-policy-core = { package= "cedar-policy-core", git = "https://github.com/cedar-policy/cedar.git", branch = "main", version = "=4.1.0"}
-cedar-policy-validator = { package= "cedar-policy-validator", git = "https://github.com/cedar-policy/cedar.git", branch = "main", version = "=4.1.0"}
+cedar-policy = { version = "=4.3.0", path = "../cedar-policy" }
+cedar-policy-formatter = { version = "=4.3.0", path = "../cedar-policy-formatter" }
+cedar-policy-core = { package= "cedar-policy-core", git = "https://github.com/cedar-policy/cedar.git", branch = "main", version = "4.*"}
```

Comment:
Why does this change to `4.*`? (Also, why does the cli need to depend on core at all)

Reply:
It can't say 4.3.0 because current git `main` is not 4.3.0.  An alternate plan would be to move this from a git to a path dependency, like the dependency on `cedar-policy`.

CLI dependency on core was added in #1326 and is only necessary for the protobufs-related features.  Now that we have #1345, I suspect this is no longer needed, and I'm working on removing it.

Reply:
Perhaps related that Cargo.lock has two entries for `cedar-policy-core` at different commits.

Reply:
Oof. Looking closer, #1326 also added some JSON-related commands that are not guarded by the protobufs experimental flag, and rely on the undocumented JSON serialization of internal structures (which is what Lean expects as of this writing).  Be nice to remove those when Lean switches to accepting protobuf instead, but technically that's a breaking change to functionality not marked as experimental

Reply:
Same goes for `cedar-policy-validator` (one line below). Can we use path dependencies for both?

Reply:
> Be nice to remove those when Lean switches to accepting protobuf instead, but technically that's a breaking change to functionality not marked as experimental

All unrelased code, so please break

Reply:
Done